<!DOCTYPE html>
<html lang="en"/>
<link href="css/style.css" rel="stylesheet" type="text/css" />

<head>
	<title>Experiment 16, 17 and 18</title>
	<div class="dropdown" style="position: absolute; top: 0px">
	<button style="background-color: green">Jump to: &#8659;</button>
	  <div id="myDropdown" class="dropdown-content">
	    <a href="front_cover.html">Cover</a>
		<a href="guidelines.html">Guidelines</a>
		<a href="welcome.html">Welcome</a>
	    <a href="intro.html">Introduction</a>
	    <a href="results.html">Results</a>
	    <a href="typed_reports.html">Typed reports</a>
	    <a href="discussion.html">Discussion</a>
		<a href="references.html">References</a>
		<a href="pracs.html">Prac list</a>
	    <a href="exp1.html">Experiment 1</a>
	    <a href="exp2.html">Experiment 2</a>
	    <a href="exp3.html">Experiment 3</a>
	    <a href="exp4.html">Experiment 4</a>
	    <a href="exp5.html">Experiment 5</a>
	    <a href="exp6.html">Experiment 6</a>
	    <a href="exp7.html">Experiment 7</a>
	    <a href="exp8.html">Experiment 8</a>
	    <a href="exp9.html">Experiment 9</a>
	    <a href="exp10.html">Experiment 10</a>
	    <a href="exp11.html">Experiment 11</a>
	    <a href="exp12.html">Experiment 12</a>
	    <a href="exp13.html">Experiment 13</a>
	    <a href="exp14.html">Experiment 14</a>
	    <a href="exp15.html">Experiment 15</a>
	    <span style="background-color: green"><a:active href="exp16.html">Experiment 16</a></a:active></span>
	    <a href="exp16.html">Experiment 17</a>
	    <a href="exp16.html">Experiment 18</a>
	    <a href="exp19.html">Experiment 19</a>
	    <a href="exp20.html">Experiment 20</a>
	    <a href="exp21.html">Experiment 21</a>
	    <a href="exp22.html">Experiment 22</a>
	    <a href="exp23.html">Experiment 23</a>
	    <a href="exp24.html">Experiment 24</a>
	  </div>
</div>
</head>
<body>

<h2>
	<pre style="text-align: left;">
	<big>
HBC 106

Experiments 16, 17 and 18

ISOLATION OF DNA FROM BACTERIA
	</big>
	</pre>
</h2>

<body>
The biological role of nucleic acids.<br/>
This field has grown rapidly in recent years and it is not possiblem to do justice to the topic in only a few words.  A brief moutline however seems appropriate at this point.
<b>DNA</b><br/>
<em>Occurrence:</em> DNA is intimately associateed with the genetic material of the cell.  In the some organisms a single strand of DNA seems to be nthe store of the genetic inforation, but in higher mnorganisms, the DNA is present as nucleoproteins in the chromosomes.  The amount ofn DNA in the cell of a nparticular species is constant. whereas the germ cells, with half the number of chromosomes, contain half the amount of DNA present  in other cells.  The DNA occurs almost exclusively in the nucleus with trace amounts in the mitochondria and chloroplast.<br/>

<em>Function:</em>  The main function of DNA is to act as a store of genetic information.  Hereditary characteristics are passed on to daughter cells through replication of DNA which appears to take place in a series of short nsteps as segments of the double helix are unwound and the new DNA is formed on each strand simultaneously from free nucleotides.  The newly synthesized strands are built up according to the rulesn on base pairing so that neventually two identical molecules are formed from the original DNA.<b>DNA</b> is also the template for synthesis of proteins in the cell with a triplet nof three bases providing the genetic code for each amino acid.<br/>
<b>RNA</b><br/> 
Whereas DNA is found almost exclusively in the nucleus, RNA is found distributed throughout out the cell.  Most of the RNA is present in the cytoplasm as soluble and ribosomal RNA, but about 10% is found in the nucleus with trace also present in the mitochondria.  The number of RNAs present in the cells of higher organisms has gone from the three commonly known RNAs: messenger RNA (mRNA), ribosomal RNA (rRNA), and transfer RNA (tRNH all of which are actively involved in the synthesis of proteins), to nearly seven depending on how you classify them.  These will be discussed in more details in class.Protein synthesis
The genetic message is first passed from the nuclear DNA to mRNA through the bases; a process known as transcription.  The mRNA then migrates into nthen cytoplasm and it is here in association with the ribosomes and tRNA thatn the four letter base code is translated into 20-letter amino acid code of proteins: a process known as translation.  Each amino acid has its won tRNA whose function is to transfer the nactivated amino acid to the site of protein synthesis.
Nucleic acids of viruses.<br/>
All viruses contain nucleic acids wihich may account for as much as 50 per cent of the particle.  The nucleic acid may single or double stranded and may be present as a linear or a circular molocule.  The nucleic acid has a well-defined tertiary structure and  is surrounded by a protective coat of protein.  The nucleic acid is the infective part of the virus while the protein accounts for it immunological specificity.  The virus attaches its self to the cell and injects its own DNA into the host, thus directing the infected cell to synthesize virus proteins and nucleic acids in order to multiply.  Most plan viruses contain RNA although a few contain DNA.  In this case the RNA acts as both the genetic material as well as a template for protein synthesis.

Materials:
<ol type="1">
	<li>E. Coli suspension (Luria Bertain broth. 37<sup>o</sup>)C overnight) </li>
	<li>1% Sodium dodecyl sulphate (SDS)</li>
	<li>Test tubes. 50 ml Falcon tubes, sterile beakers (50 ml)</li>
	<li>1.5ml Eppendorf tubes</li>
	<li>Water bath (60-65<sup>o</sup>)C</li>
	<li>Stirring rod (sterile)</li>
	<li>95% Ethanol (chilled)</li>
	<li>Dropper</li>
	<li>Blue and yellow tips (sterile/autoclaved)</li>
	<li>Agarose</li>
	<li>10 x TBE buffer (Tris. Boric Acid and EDTA)</li>
</ol>

Procedure
<ol type="1">
	<li>Add 5 ml bacterial suspension to a test tube </li>
	<li>Add 1 ml of nSDS to the test tube with bacteria, gently rotate and swirl to mix in SDS (approx 5 min).</li>
	<li>Stand tube in 60-65<sup>o</sup>)C< water bath for 30 min./li>
	<li>Allow to cool to room temperature.</li>
	<li>Place a striring rod into the test tube and add m2ml of cold 95% ethanol slowly with the dropper down the string rod into the tube.</li>
	<li>Hold tube at a 45 degree angle and slowly rotate nyour stirring road clockwise (avoid touching then sides of the tube). approx 5 min. /li>
	<li>Remove rod and immerse rod with bacterial DNA in 95% ethanol for approx. 2 min.</li>
	<li>Remove the alcohol, air dry forn 10 min. then suspend DNA in sterile water.</li>
	<li>Load samples on  0.7% agarose gel and run at 80 V for 1 hr.</li>
</ol>

Process
<ol type="1">
	<li>DNA isolation on Day 1 </li>
	<li>DNA characterization by UV spectroscopy, Day 2</li>
	<li>DNA characterization by agarose gel electrophoresis Day 2.</li>
	<li>DNA digestion with Restriction enzymes and characterization by agarose gel electrophoresis, Day 3</li>

Agarose gel electrophoresis of DNA (Day 2 & 3)
In order to be able to visualize the genomic DNA an aliquot of the DNA will be run on agarose gel and visualized by UV transillumination.  To effect a size estimate one must remember to run a wild type lambda DNA that has been digested with Hind III alongside the genomic DNA.
<ul type="circle">
	<li>Heat  0.8% agarose in 1 XBTE mixture on a stirring hotplate until completely liquefied.  Add lug/ml EtBr and swirl to mix.</li>
	<li>Cool until tolerable to hold.   Pour the agarose into the gel apparatus to a depth of about 1 cm.</li>
	<li>Place the comb into the comb holder and allow the gel to sit undisturbed until coll and solid.</li>
	<li>Remove the gates/gel forming plates at either end of the gel and pour suffient 1 XTBE into the well to completely cover the gel.</li>
	<li>Slowly and carefully remove the comb.  Be careful not to pull the bottoms of the wells out with the comb or your sample will diffuse under the gel and get lost.  The well should fill with buffer.</li>
	<li>Place 3 &micro; of loading dye in each genomic DNA sample. Mix</li>
	<li>Remove the entire contents of each tube with a pipepttman and slowly drop the contents of the pipette into the well of the gel.  Be sure to load the Lambda HinaIII size marker in the first lane of the gel.  When loading into the wells it is important not to poke the tip of the pipette into the front wall of th wellthropught which the DNA will pass.  This causes distorted banding patterns to develop.  So aim the pipette tip towards the back wall of th well.</li>
	<li>Place the cover over the apparatus and run the gel at 150 volts.  Be sure the electrode farthest from the wells is the red (+) positive electrode your samples will run backwards.  Run the gel until the dye marker is <big>&frac34;</big> of the way through the gel.</li>

Visualizing the DNA
Turn off the power and remove the gel cover.  Be aware that sufficient current is running through these gels to KILL a human being! Be conscius of what you are doing. 
</ul>
	</ol>
<ul type="circle">
	<li>Pour off the buffer. This buffer already contains the fluorescent dye Ethidium Bromide (Et Br) which causes the DNA bands to become visible under the UV+ light.</li>
	<li>Be certain to handle these gels with gloves as the Et Br is powerful mutagen and suspected cancer causing agent. </li>
	<li>Place the gell on the UV light box and view the digested material. </li>
	<li>Photograph any good results. </li>
</ul>

Digestion of Genomic DNA  with restriction enzymes<br/>
You will be provided with a tube containing ~10 ug undigested genomic DNA. Carry out the digestion as follows:<br/>
To a tube
<ul type="circle">
	<li>17 &micro;l of DNA</li>
	<li>2 &micro;l 10<big>&times;</big> Enzyme buffer</li>
	<li>1 &micro;l of Enzyme(~10 U/&micro;l</li>
</ul>

Incubate at 37<sup>o</sup>C overnight.<br/>
Add 12 &micro;l of 6 <big>&micro;</big> loading dye and mix thoroughly.<br/>
Load on a 0.8% agarose gel and electrophorese at 150 volts <big>&times;</big> 2 hr as before.<br/>
Visualize and photograph any good results.

<ul type="none">
	<li>Estimating the size of the DNA</li>
	<li>To estimate the size of DNA it</li>
	<li>is necessary to first develop a</li>
	<li>standard curve of migration</li>
	<li>rates from the Lambda</li>
	<li>HindIII DNA fragments that</li>
	<li>you have run on your gel.</li>
	<li>The sizes of the lambda</li>
	<li>fragments are as follows:</li>
</ul>
&emsp;&emsp;&emsp;&emsp;&emsp;Migration distance
<ul type="none">
	<li>Fragment size in bps</li>
	<li>23, 130</li>
	<li>9, 416</li>
	<li>6, 557</li>
	<li>4, 361</li>
	<li>2, 322</li>
	<li>2, 027</li>
	<li>564</li>
</ul>

The smallest two fragments will probably not be resolvable on the gells that you are running, so only six distinct bands should be visible.<br/>
To develop a standard curve, measure with a ruler the distance of migration of each of the bands from the well. Then draw a graph of these results with migration distance on one axis and fragement size of the DNA bands/fragments it is only necessary to measure the distance of migration of the particular fragement from its well and locate the migration distance on the standard curve. Nowadays these sorts of calculations are done for you on the computer either using spreadsheet programs of dedicated DNA analysis programs.


<p>
	NB: Preparation of 10 <big>&times;</big> TBE (Tris-Boric acid-EDTA)<br/>
	Trizma Base, 108g<br/>
	Boric acid, 55g<br/>
	0.25 M EDTA, 40 ml<br/>
	Bring up to 1 litre
</p>

<p>
	References<br/>
	<ol type="1">
		<li> An introduction to practical biochemistry. D.T. Plummer. Mcgraw-Hill Book Company, UK. 1987. pp 332</li>
	</ol>
</p>
</body>
</html>